rosbridge_library/src/rosbridge_library/capabilities/call_service.py

```diff
@@ -52,7 +53,10 @@ def __init__(self, protocol):
         Capability.__init__(self, protocol)
 
         # Register the operations that this capability provides
-        protocol.register_operation("call_service", self.call_service)
+        # This calls the service in a separate thread so multiple services can be processed simultaneously.
+        protocol.register_operation(
+            "call_service", lambda msg: Thread(target=self.call_service, args=(msg,)).start()
```

Comment:
```suggestion
            "call_service", lambda msg: Thread(target=self.call_service, args=[msg]).start()
```
Not sure if it's valid, but I find this more readable